README.md

```diff
@@ -61,6 +59,13 @@ If you want to add a source, you only have to implement the `FetcherInterface`.
 
 Some fetchers will use tags to fetch their data, and some of these tags may be missing. If this is the case, you should find the fetcher in `src\Fetcher` and add the missing tag to its tags list.
 
+## Slack
+
+Starfleet uses strong implementation with Slack as it sends daily Slack notifications. You must [create a Slack application](https://api.slack.com/apps) if you don't have any.
+You need to configure a webhook at `https://[your-slack-organization].slack.com/apps/A0F7XDUAZ-incoming-webhooks` and add it to the SLACK_WEB_HOOK_URL env variable.
+Since Starfleet uses user interaction, you must [configure your application](https://api.slack.com/apps/A01V000RWFJ/interactive-messages?) to allow user interaction. Your Slack Signing Secret should be stored in the SLACK_SIGNING_SECRET env variable.
```

Comment:
```suggestion
Since Starfleet uses user interaction, you must [configure your application](https://api.slack.com/apps/A01V000RWFJ/interactive-messages?) to allow user interaction. Your Slack Signing Secret should be stored in the `SLACK_SIGNING_SECRET` env variable.
```